src/Event/PostReadEvent.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace ApiPlatform\Core\Event;
+
+use ApiPlatform\Core\DataProvider\CollectionDataProviderInterface;
+use ApiPlatform\Core\DataProvider\ItemDataProviderInterface;
+use ApiPlatform\Core\DataProvider\SubresourceDataProviderInterface;
+use ApiPlatform\Core\Identifier\IdentifierConverterInterface;
+use ApiPlatform\Core\Serializer\SerializerContextBuilderInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+class PostReadEvent extends Event
```

Comment:
```suggestion
final class PostReadEvent extends Event
```

We mark all classes final by default